calico-enterprise/_includes/components/ReqsSys.js

```diff
@@ -29,9 +29,11 @@ function NodeRequirementsEnt(props) {
           <ul>
             {(props.orch === orchestrators.Kubernetes || props.orch === orchestrators.HostProtection) && (
               <>
-                <li>CentOS 8</li>
```

Comment:
[CentOS 8 is EoL'ed](https://www.centos.org/centos-linux-eol/). CentOS (stream) in general is just one of the RHEL related or derived OSes. I would like to categorize it into the RHEL "binary-compatible operating systems".